transport/control.go

```diff
@@ -46,8 +46,8 @@ const (
 	// The default value of flow control window size in HTTP2 spec.
 	defaultWindowSize = 65535
 	// The initial window size for flow control.
-	initialWindowSize             = defaultWindowSize      // for an RPC
-	initialConnWindowSize         = defaultWindowSize * 16 // for a connection
+	initialWindowSize             = defaultWindowSize * 2048      // for an RPC
+	initialConnWindowSize         = defaultWindowSize * 16 * 2048 // for a connection
```

Comment:
> How would latency for small requests sharing the same connection be hurt by a bigger window; I don't see how their frames would be delayed since they are likely to be sent and reach the receiver sooner.

Ah, ignore me. I misremembered how framing when writing data to a connection works. See `http2Client.Write`. Data is interleaved in 16 KB chunks regardless of the stream/connection window sizes.